src/templates/style-imports.js

```diff
@@ -0,0 +1,17 @@
+const { stylingTypes } = require('../helpers');
+
+module.exports = (style, componentName) => {
+  switch (style) {
+    case stylingTypes.STYLED:
+      return "import styled from 'styled-components';";
+
+    case stylingTypes.CSS_MODULES:
+      return `import styles from './${componentName}.css';`;
+
+    case stylingTypes.APHRODITE:
+      return "import { StyleSheet, css } from 'aphrodite';";
+
+    default:
+      return '';
+  }
```

Comment:
If we add `none` as a valid styling type, I would just do this:

```js

case stylingTypes.NONE:
default:
  return '';
```

By using the fallthrough we can make `none` do the default thing, but we're being explicit about it.

Reply:
Sounds good. This (the fallthrough) was the reason I suggested using `switch` statements, neat!